tests/clients/cli/wrapper_cli.go

```diff
@@ -60,8 +59,8 @@ func (w *cliWrapper) executeStream(ctx context.Context, args []string) (io.ReadC
 		args = append(args, "--tx", fmt.Sprintf("%d", tx.ID()))
 	}
 	id := db.GetContextIdentity(ctx)
```

Comment:
Since identity is a struct it will always have a default value. In this case we only care about the private key being non nil.

Reply:
the point of the `Option` is to express the zero/empty value explicitly, rather than implicitly with Go zero values (like a default struct).

If we're going to wrap the identity in the `Option` type, we should at least use it correctly.